Packs/SecureWorks/ReleaseNotes/5_0_3.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+
+##### TaegisXDR v2
+
+- Added support for *serviceDeskId*, *serviceDeskType*, and *tags* for arguments in ***taegis-update-investigation*** command.
+- Added support for *assignee_user*, *alertsEvidence*, and *tags* when fetching incidents/investigations.
```

Comment:
I didn't see any specifications for this in the yml file or the readme. 
Does this change only takes place in the query string?

Reply:
Yup, those are only for when querying data. The command `taegis-add-evidence-to-investigation` allows for adding alertsEvidence to an invesitgation.